src/main/java/com/arcbees/bourseje/client/admin/dashboard/AdminDashboardPresenter.java

```diff
@@ -132,4 +149,23 @@ public void onSuccess(VoteState currentState) {
             }
         });
     }
+
+    private Map<String, Integer> convertToMap(List<Candidate> candidatesList, Collection<CandidateResult> candidateResults) {
```

Comment:
I guess this whole method will be removed when the candidates are going to be fetched from the server?